### DISCUSSION TITLE
[In the distant future Suggestion] External Model "Import" Workaround

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#4023] [Suggestions]
A number of new players have asked if there is a ship import feature where you are able to import another build from a 3d model/other voxel which exists in one form or another in Starmade, Avorion, Minecraft, and I assume others. The answer is obviously no, but with the extra assumption that it won't be added because not only do the above don't work the best and in some cases can be more trouble trying to fix oddly imported builds then just doing it on your own, but with bricks and stretch tech chances are imported builds would be even more trouble. All that being said, being able to use a reference, usually a 3d model, would be a great distant feature.

A workaround would be rather than trying to automatically convert a 3d model into SEVO voxel, how about in a shipyard you can load a translucent (holographic) model of the ship. Translucent and incorporeal, the player can build in/around it giving the reference they need without needing to add some complicated .stl to .mps feature. 
![image](https://user-images.githubusercontent.com/36980446/106681187-3e91ec00-657d-11eb-8549-4e3d9c382a62.png)
Something similar is the projector from Space Engineers, but from what I see that is only for already built ships. This feature would be much more powerful allowing the building in/around "any" imported model.

Of course this would be something to consider in the distant future, but this would be a great assist to builders that want to build great replicas of existing models or ones they make themselves.

**Comment by Zerefette** [comment]
It would be nice if you could add the file, assign textures and largest dimension and project.
